### PR TITLE
Add ArgoCD wave number to plan CRD object

### PR DESCRIPTION
**Pull request by megian**
ArgoCD must deploy the plan CRD after set up all SUC components.
This ensures the object kind plan.upgrade.cattle.io/v1
is known to the cluster.

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the ./CHANGELOG.md.

### CODE REVIEW

**Review by Kidswiss — APPROVED** [review]

**Comment by Kidswiss** [comment]
LGTM

File: lib/suc.libjsonnet [hunk 1]
```diff
@@ -7,6 +7,9 @@ local params = inv.parameters.system_upgrade_controller;
   Plan(name, channel, label_selectors, concurrency, image, push_gateway, command): kube._Object('upgrade.cattle.io/v1', 'Plan', name) {
     metadata+: {
       namespace: params.namespace,
+      annotations+: {
+        'argocd.argoproj.io/sync-wave': '5',
```
Comment (simu) [inline lib/suc.libjsonnet:11]: How was '5' chosen as the value? Just curious if we already have defined a set of sync-waves to use.

**Review by simu — APPROVED** [review]

File: lib/suc.libjsonnet (hunk 1 above)
Comment (megian) [reply lib/suc.libjsonnet:11]: Well did just choose something which still leave some place for other waves if required. Just like the good old BASIC method of keep some numbers free for further use.

To my knowledge, we haven't defined overall rules yet.

Comment (simu) [reply lib/suc.libjsonnet:11]: All good for me, was just curious

**Pull request merged**